scripts/monitor_ci.js

```diff
@@ -12,8 +12,10 @@ const fetch = require('node-fetch');
 const util = require('util');
 const log = console.log;
 
-// TODO: Check if multiple repositories and toggle by staging or master branch
-const root = 'https://contractrepostaging.komputing.org/';
+const root = (process.env.CIRCLE_BRANCH === 'staging')
+  ? 'https://contractrepostaging.komputing.org/'
```

Comment:
Shouldn't be easier here to just use: process.env.REPOSITORY_URL instead of the logic?

Reply:
@edisinovcic Is it visible here though? We're not in a docker container when running this script, only in Circle.

Reply:
@edisinovcic  I'm going to merge this just to see if works on staging (because it probably won't ha ha!!). Will revisit this question...

Reply:
It looks like it worked actually...

https://circleci.com/gh/ethereum/source-verify/1201

Deployments to Goerli via Infura/Truffle are a little flaky though - have seen it fail a couple times with a `contract couldn't be stored` error. The job has to be re-run. 

I think this is because of a [long-standing issue][1] with some clients (Parity?) returning a transaction receipt before they've successfully stored the bytecode, causing a race-condition crash in Web3 1.2.1.

Will look at using something besides truffle migrations to deploy if this becomes too unstable/annoying.

[1]: https://github.com/trufflesuite/truffle/issues/2257

Reply:
Sorry, just now received a notification. Environment could be loaded, but if this works this way then it's good. :) Maybe adding todo to check this later could be useful.

Reply:
I have to get up and running full nodes for goerli and rinkeby on server and we can then use them for this instead of infura?

There is still some config there that I would like to add. Also need to do stop geth for mainnet to resize the server, but I'm not sure if that will corrupt the database... Anyways we could have this fixed in relatively short time. We could just create an issue for it so we know to address it in the future.

Reply:
> I have to get up and running full nodes for goerli and rinkeby on server and we can then use them for this instead of infura?

Oh yeah that would be awesome! I'll make an issue for this and for the env thing.